index.html

```diff
@@ -3211,7 +3211,22 @@ <h3>Reserved Extension Points</h3>
               <th>Description</th>
             </tr>
           </thead>
+
           <tbody>
+            <tr>
+              <td>`confidenceMethod`</td>
```

Comment:
```suggestion
              <td>`confidenceMethod`</td>
               <td>`confirmationMethod`</td>
```

Reply:
-1 to bringing back `confirmationMethod`.

Reply:
how would you suggest we map `cnf` then?

Reply:
@mprorock,

> how would you suggest we map `cnf` then?

A specific, single-key-style proof-of-possession type of `confidenceMethod` (which someone will need to define) can be mapped to `cnf`. Other types clearly wouldn't map, for example:

```
{
  ...,
  "confidenceMethod": [{
    "type": "FooSingleKeyProofOfPossessionConfirmationThing",
    "cnf": { /* same kind of stuff that goes in 'cnf' }
  }, {
    "type": "DoesNotWorkWithCnf",
    ...
  }]
}
```